Fitness/Backend/docker-compose.yml

```diff
@@ -1,31 +1,56 @@
 services:
```

Comment:
Generally, I think this is not necessary. Why? Because, the (main) docker-compose.yaml file is used to define which services are needed (containers, their images and respective Dockerfiles etc.). Maybe we could add the dependencies (but we need to think this through), but the configuration (i.e. which ports are exposed, or how the volume mapping is configured) is specific to the environment on which the application will be run on (for example, we will not have the same configuration for development (on local machines) as on AWS or some other cloud solution. My proposal is the following:

1. Rename `docker-compose.override.yaml` to `docker-compose.development.yaml` to indicate that the file is used in development stage.
2. Move dependencies from `docker-compose.override.yaml` to `docker-compose.yaml` as noted.